libs/vkd3d/command.c

```diff
@@ -1857,39 +1857,46 @@ static VkImageAspectFlags vk_writable_aspects_from_image_layout(VkImageLayout vk
     }
 }
 
-static int d3d12_command_list_find_attachment(struct d3d12_command_list *list,
-        const struct d3d12_resource *resource, const struct vkd3d_view *view)
+static int d3d12_command_list_find_attachment_view(struct d3d12_command_list *list,
+        const struct d3d12_resource *resource, const VkImageSubresourceLayers *subresource)
 {
     unsigned int i;
 
     if (list->dsv.resource == resource)
     {
-        const struct vkd3d_view *dsv = list->dsv.view;
+        struct vkd3d_view *dsv = list->dsv.view;
```

Comment:
Why drop the const?

Reply:
Re-added; this was part of a temporary change and is no longer needed.